capstone-backend/pom.xml

```diff
@@ -43,30 +43,15 @@
 
   <build>
     <plugins>
-      <!-- Old plugin that is deprecated for deployment but can be used for local testing. 
-      Enables usage of `mvn appengine:devserver` for local testing, which is in many of the walkthroughs. 
-      Cannot deploy with `mvn appengine:update` because it's deprecated. -->
       <plugin>
         <groupId>com.google.cloud.tools</groupId>
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <deploy.projectId>stevensolar-step2020</deploy.projectId>
+          <deploy.projectId>mauroguerrero-step2020</deploy.projectId>
```

Comment:
Was this for you to test as you built or should we just deploy under your account from here on out? Either is fine with me just wanted to make sure you intended to leave this here in the PR.